licenses/mit_license.html

```diff
@@ -0,0 +1,372 @@
+<!DOCTYPE html>
```

Comment:
required license? or should it be deleted?

Reply:
Due to changing our dependencies, we don't auto-gen the `mit_license.txt` anymore, only the `.html` so I've changed it to stay consistent with others